cosmos/gaia
# feat(e2e): add e2e tests for the feegrant module

close #1675

## Description

Add tests for the `feegrant` module.

### Test workflow

- Create a fee grant with Alice as a `granter` and Bob as a `grantee.`
- Send tx from Bob to a random address passing Alice as a `fee-granter` flag using all feegrant coins allowed;
- Send tx from Bob to a random address should fail because the `spend-limit` was reached;
